### ISSUE TITLE
Could not build the application for the simulator. Error launching application on iPhone 13 Pro Max

### ISSUE THREAD

**Issue by bholendraofficial** [flutter/flutter#91958]
<details>
<summary>console error log</summary>

```

Launching lib/main.dart on iPhone 13 Pro Max in debug mode...
Running pod install...
Running Xcode build...
Xcode build done.                                           10.5s
Failed to build iOS app
Error output from Xcode build:
↳
    --- xcodebuild: WARNING: Using the first of multiple matching destinations:
    { platform:iOS Simulator, id:dvtdevice-DVTiOSDeviceSimulatorPlaceholder-iphonesimulator:placeholder, name:Any iOS Simulator Device }
    { platform:iOS Simulator, id:1C9DD1E4-ECDF-42EB-AAC9-CC7C31277559, OS:15.0, name:iPad (9th generation) }
    { platform:iOS Simulator, id:6BA103EB-529E-4E21-AED5-D52B7CAAB356, OS:15.0, name:iPad Air (4th generation) }
    { platform:iOS Simulator, id:EC412ADC-D453-4FAB-9D4A-C773B206A2AB, OS:15.0, name:iPad Pro (9.7-inch) }
    { platform:iOS Simulator, id:3C653181-3D42-42EF-AAAB-89F83EC9CF11, OS:15.0, name:iPad Pro (11-inch) (3rd generation) }
    { platform:iOS Simulator, id:6EF444B5-2D77-45D0-A9E9-E13B4CF48973, OS:15.0, name:iPad Pro (12.9-inch) (5th generation) }
    { platform:iOS Simulator, id:9078AA2A-1437-48EB-953A-FF7E6269A472, OS:15.0, name:iPad mini (6th generation) }
    { platform:iOS Simulator, id:6AF1FBF7-D549-4C76-BB6F-51B3D3E9B626, OS:15.0, name:iPhone 8 }
    { platform:iOS Simulator, id:718E790E-1A25-4811-8943-2BB78A65569E, OS:15.0, name:iPhone 8 Plus }
    { platform:iOS Simulator, id:BE96BF05-5564-4B34-920E-B4EBE7961B67, OS:15.0, name:iPhone 11 }
    { platform:iOS Simulator, id:BF6A7308-D1A0-42D1-8D37-5FC238D82771, OS:15.0, name:iPhone 11 Pro }
    { platform:iOS Simulator, id:2EFE52E7-4317-4D94-9E89-C30E0B3116AA, OS:15.0, name:iPhone 11 Pro Max }
    { platform:iOS Simulator, id:F548CEC1-E4B9-4FBB-BC5C-3A17FA359D10, OS:15.0, name:iPhone 12 }
    { platform:iOS Simulator, id:988693B2-840F-43BA-A657-84F7BF145D17, OS:15.0, name:iPhone 12 Pro }
    { platform:iOS Simulator, id:5D80F556-5D51-4258-A179-B1C96F3466F4, OS:15.0, name:iPhone 12 Pro Max }
    { platform:iOS Simulator, id:86B309C5-371F-48EC-842F-7B33A8E229F1, OS:15.0, name:iPhone 12 mini }
    { platform:iOS Simulator, id:DDD08932-1628-478F-820D-576486D4D592, OS:15.0, name:iPhone 13 }
    { platform:iOS Simulator, id:4F1A97B3-B9EE-4C87-BF03-D1FA9EAF71DC, OS:15.0, name:iPhone 13 Pro }
    { platform:iOS Simulator, id:A30A16F1-EF35-44F4-A360-71237C35951F, OS:15.0, name:iPhone 13 Pro Max }
    { platform:iOS Simulator, id:677791F0-6D08-4404-ABE3-828CD0F5704A, OS:15.0, name:iPhone 13 mini }
    { platform:iOS Simulator, id:3E062C4E-3DE5-4C0B-B57F-17122CD5BECC, OS:15.0, name:iPhone SE (2nd generation) }
    { platform:iOS Simulator, id:72F91FA0-D491-45A9-936A-6B5917BF3F20, OS:15.0, name:iPod touch (7th generation) }
    { platform:macOS, arch:arm64, variant:Mac Catalyst, id:00008103-001965E826D9001E }
    { platform:macOS, arch:x86_64, variant:Mac Catalyst, id:00008103-001965E826D9001E }
    { platform:macOS, arch:arm64, variant:Designed for [iPad,iPhone], id:00008103-001965E826D9001E }
    { platform:iOS, id:dvtdevice-DVTiPhonePlaceholder-iphoneos:placeholder, name:Any iOS Device }
    { platform:macOS, variant:Mac Catalyst, name:Any Mac }
    ** BUILD FAILED **


Xcode's output:
↳
    Command CompileSwiftSources failed with a nonzero exit code
    /Users/bholendraofficial/Library/Android/flutter/.pub-cache/hosted/pub.dartlang.org/jitsi_meet-4.0.0/ios/Classes/JitsiViewController.swift:8:41: error: cannot find type 'PiPViewCoordinator' in scope
        fileprivate var pipViewCoordinator: PiPViewCoordinator?
                                            ^~~~~~~~~~~~~~~~~~
    /Users/bholendraofficial/Library/Android/flutter/.pub-cache/hosted/pub.dartlang.org/jitsi_meet-4.0.0/ios/Classes/JitsiViewController.swift:8:41: error: cannot find type 'PiPViewCoordinator' in scope
        fileprivate var pipViewCoordinator: PiPViewCoordinator?
                                            ^~~~~~~~~~~~~~~~~~
    /Users/bholendraofficial/Library/Android/flutter/.pub-cache/hosted/pub.dartlang.org/jitsi_meet-4.0.0/ios/Classes/JitsiViewController.swift:87:30: error: cannot find 'PiPViewCoordinator' in scope
            pipViewCoordinator = PiPViewCoordinator(withView: jitsiMeetView)
                                 ^~~~~~~~~~~~~~~~~~
    /Users/bholendraofficial/Library/Android/flutter/.pub-cache/hosted/pub.dartlang.org/jitsi_meet-4.0.0/ios/Classes/JitsiViewController.swift:102:30: error: 'nil' requires a contextual type
            pipViewCoordinator = nil
                                 ^
    note: Using new build system
    note: Planning
    note: Build preparation complete
    note: Building targets in parallel
    note: Removed stale file '/Users/bholendraofficial/Library/Developer/Xcode/DerivedData/Runner-bujcctrsjypcpcbrdtogiplpmxwv/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/PromisesObjC.build/Script-006B616305AB878E0204F79A5A0DA976.sh'

    warning: Stale file '/Users/bholendraofficial/Documents/StudioProjects/connect/build/ios/Debug-iphonesimulator/Runner.app/Frameworks/FirebaseABTesting.framework' is located outside of the allowed root paths.

    warning: Stale file '/Users/bholendraofficial/Documents/StudioProjects/connect/build/ios/Debug-iphonesimulator/Runner.app/Frameworks/FirebaseCrashlytics.framework' is located outside of the allowed root paths.

    warning: Stale file '/Users/bholendraofficial/Documents/StudioProjects/connect/build/ios/Debug-iphonesimulator/Runner.app/Frameworks/FirebaseInstallations.framework' is located outside of the allowed root paths.

    warning: Stale file '/Users/bholendraofficial/Documents/StudioProjects/connect/build/ios/Debug-iphonesimulator/Runner.app/Frameworks/FirebasePerformance.framework' is located outside of the allowed root paths.

    warning: Stale file '/Users/bholendraofficial/Documents/StudioProjects/connect/build/ios/Debug-iphonesimulator/Runner.app/Frameworks/FirebaseRemoteConfig.framework' is located outside of the allowed root paths.

    warning: Stale file '/Users/bholendraofficial/Documents/StudioProjects/connect/build/ios/Debug-iphonesimulator/GoogleUtilities/GoogleUtilities.framework/Headers/GULObjectSwizzler.h' is located outside of the allowed root paths.

    warning: Stale file '/Users/bholendraofficial/Documents/StudioProjects/connect/build/ios/Debug-iphonesimulator/GoogleUtilities/GoogleUtilities.framework/Headers/GULOriginalIMPConvenienceMacros.h' is located outside of the allowed root paths.

    warning: Stale file '/Users/bholendraofficial/Documents/StudioProjects/connect/build/ios/Debug-iphonesimulator/GoogleUtilities/GoogleUtilities.framework/Headers/GULSwizzledObject.h' is located outside of the allowed root paths.

    warning: Stale file '/Users/bholendraofficial/Documents/StudioProjects/connect/build/ios/Debug-iphonesimulator/GoogleUtilities/GoogleUtilities.framework/Headers/GULSwizzler.h' is located outside of the allowed root paths.

    warning: Stale file '/Users/bholendraofficial/Documents/StudioProjects/connect/build/ios/Debug-iphonesimulator/GoogleUtilities/GoogleUtilities.framework/Headers/GULUserDefaults.h' is located outside of the allowed root paths.

    /Users/bholendraofficial/Documents/StudioProjects/connect/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 15.0.99. (in target 'libPhoneNumber-iOS' from project 'Pods')
    /Users/bholendraofficial/Documents/StudioProjects/connect/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 15.0.99. (in target 'AppAuth' from project 'Pods')
    /Users/bholendraofficial/Documents/StudioProjects/connect/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 15.0.99. (in target 'Reachability' from project 'Pods')
    /Users/bholendraofficial/Documents/StudioProjects/connect/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 15.0.99. (in target 'BoringSSL-GRPC' from project 'Pods')
    /Users/bholendraofficial/Documents/StudioProjects/connect/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 15.0.99. (in target 'abseil' from project 'Pods')
    /Users/bholendraofficial/Documents/StudioProjects/connect/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 15.0.99. (in target 'gRPC-C++' from project 'Pods')
    /Users/bholendraofficial/Documents/StudioProjects/connect/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 15.0.99. (in target 'gRPC-C++-gRPCCertificates-Cpp' from project 'Pods')
    /Users/bholendraofficial/Documents/StudioProjects/connect/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 15.0.99. (in target 'GoogleSignIn' from project 'Pods')
    /Users/bholendraofficial/Documents/StudioProjects/connect/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 15.0.99. (in target 'gRPC-Core' from project 'Pods')
    /Users/bholendraofficial/Documents/StudioProjects/connect/ios/Pods/Pods.xcodeproj: warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 15.0.99. (in target 'leveldb-library' from project 'Pods')

Could not build the application for the simulator.
Error launching application on iPhone 13 Pro Max.



```

</details>

```
[✓] Flutter (Channel stable, 2.5.3, on macOS 11.6 20G165 darwin-arm, locale en-IN)
    • Flutter version 2.5.3 at /Users/bholendraofficial/Library/Android/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 18116933e7 (24 hours ago), 2021-10-15 10:46:35 -0700
    • Engine revision d3ea636dc5
    • Dart version 2.14.4

[✓] Android toolchain - develop for Android devices (Android SDK version 31.0.0)
    • Android SDK at /Users/bholendraofficial/Library/Android/sdk
    • Platform android-31, build-tools 31.0.0
    • ANDROID_HOME = /Users/bholendraofficial/Library/Android/sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jre/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7249189)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 13.0, Build version 13A233
    • CocoaPods version 1.11.2

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 2020.3)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7249189)

[✓] VS Code (version 1.61.0)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.27.0

[✓] Connected device (2 available)
    • iPhone 13 Pro Max (mobile) • A30A16F1-EF35-44F4-A360-71237C35951F • ios            • com.apple.CoreSimulator.SimRuntime.iOS-15-0 (simulator)
    • Chrome (web)               • chrome                               • web-javascript • Google Chrome 94.0.4606.81

• No issues found!

```

**Comment by danagbemava-nc** [comment]
Without additional information, we are unfortunately not sure how to resolve this issue. 
We are therefore reluctantly going to close this bug for now. 
Please don't hesitate to comment on the bug if you have any more information for us; we will reopen it right away!  
Thanks for your contribution.

Could everyone who still has this problem please file a new issue with the exact description of what happens, logs, and the output of `flutter doctor -v`.
All system setups can be slightly different, so it's always better to open new issues and reference related issues.
